ipu-plugin/pkg/ipuplugin/deviceplugin.go

```diff
@@ -59,9 +62,15 @@ func discoverHostDevices() (map[string]*pb.Device, error) {
 		}
 
 		device_code := strings.TrimSpace(string(deviceCodeByte))
-
-		if device_code == deviceCode {
-			if !slices.Contains(exclude, file.Name()) {
+		if mode == types.IpuMode {
+			if device_code == deviceCode {
+				if !slices.Contains(exclude, file.Name()) {
+					devices[file.Name()] = &pb.Device{ID: file.Name(), Health: pluginapi.Healthy}
+				}
+			}
+		}
+		if mode == types.HostMode {
```

Comment:
If else maybe with the upper conditional

Reply:
Done, although not required, it can help code readability.